### PR TITLE
Add PHP 8 specific NullSafe operator mutators

### PR DESCRIPTION
**Pull request by sidz**
This PR:

- [x] Adds new **NullSafeMethodCall** and **NullSafePropertyCall** mutators
- [x] Covered by tests
- [x] Schema needs an update
- [x] Doc PR: https://github.com/infection/site/pull/202

NullSafeMethodCall Mutator:
```diff
- $object?->getObject()?->getName()
+ $object->getObject()?->getName()
```
and
```diff
- $object?->getObject()?->getName()
+ $object?->getObject()->getName()
```

NullSafePropertyCall Mutator:
```diff
- $object?->property?->name
+ $object->property?->name
```
and
```diff
- $object?->property?->name
+ $object?->property->name
```

I'm not sure how to exclude these mutators from the `schema.json` for case when Infection is executed on PHP < 8. Or maybe it won't be an issue as they won't be executed anyway due to `PHP_VERSION_ID` check.

### CODE REVIEW

File: tests/phpunit/Mutator/Operator/NullSafePropertyCallTest.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            [
+            <<<'PHP'
+<?php
+
+$class->property?->nextProperty;
+PHP,
+            <<<'PHP'
+<?php
+
+$class?->property->nextProperty;
```
Comment (maks-rafalko) [inline tests/phpunit/Mutator/Operator/NullSafePropertyCallTest.php:103]: up to you, more ideas for tests:

```php
(new SomeClass())?->property;

(new SomeClass())?->methodCall();

$class?->{$property};

$class?->{$methodCall};
```

**Review by maks-rafalko — APPROVED** [review]
👌

File: tests/phpunit/Mutator/Operator/NullSafePropertyCallTest.php (hunk 1 above)
Comment (sidz) [reply tests/phpunit/Mutator/Operator/NullSafePropertyCallTest.php:103]: done

**Comment by maks-rafalko** [comment]
Cool, thank you @sidz

**Pull request merged**